queries.sql

```diff
@@ -1,4 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT name FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Please, include the whole of 2019 in your query.  So instead of doing '2019-01-01', do '2019-31-12'.  That way, all of December from that year will be included and the return data will be correct 💪